README.md

```diff
@@ -65,6 +65,17 @@ To fetch dependencies and get cooking:
 | `test` | Runs unit and functional tests.
 | `validate` | Alias for `grunt validate-shrinkwrap` task (ignoring any errors which may be reported).
 
+### Creating Dummy Data
+
+If you just want to test something quickly with a small, known test data set:
+
+1. run `db/create_db.js` to drop and re-create the local mysql db
```

Comment:
Getting this error when running `create_db.js`

```
chronicle.server.db.VERBOSE: creating user table
chronicle.server.db.WARN: error creating user table: Error: ER_TOO_LONG_KEY: Specified key was too long; max key length is 767 bytes
chronicle.server.db.WARN: something went wrong: Error: ER_TOO_LONG_KEY: Specified key was too long; max key length is 767 bytes
```

I have a fresh install of MySQL from Homebrew (Server version: 5.6.21 Homebrew).